linkerd/protocol/http/src/main/scala/com/twitter/finagle/buoyant/linkerd/Headers.scala

```diff
@@ -11,6 +11,28 @@ import java.net.URLEncoder
 import java.util.Base64
 import scala.collection.breakOut
 
+/*
+  Mapping from linkerd header names to medallia header names.
+ */
+object MedalliaHeaders {
+  val medalliaPrefix = "X-Medallia"
```

Comment:
The standard seems to be MedalliaPrefix (first letter uppercase) for object vals.